modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/declaration/DefaultXmlArtifactDeclarationLoader.java

```diff
@@ -108,6 +110,9 @@
  */
 public class DefaultXmlArtifactDeclarationLoader implements XmlArtifactDeclarationLoader {
 
+  public static final String SET_VARIABLE = "setVariable";
```

Comment:
This doesn't seem like the right place for this constants. Suggestions are welcomed.